collectors/ebpf_process.plugin/README.md

```diff
@@ -195,3 +198,13 @@ following values: ​
 -   `return`: In this mode, Netdata also monitors the calls to function. In the `entry` mode, Netdata only traces kernel
     functions, but with `return`, Netdata also monitors the return of each function. This mode creates more charts, but
     also creates an overhead of roughly 110 nanosections for each function call.
+
+## Performance
+
+Because eBPF monitoring is complex, we are evaluating the performance of this new collector collector in various
```

Comment:
There is a word duplication here ` collector collector`

Reply:
Fixed!